src/SDK/SmartDetectors.Extensions/AlertExtensions.cs

```diff
@@ -288,6 +386,31 @@ private static IEnumerable<AlertProperty> CreateAlertProperties(object propertyO
             TablePropertyAttribute tableAttribute,
             Order order)
         {
+            if (propertyValue is PropertyReference propertyReferenceValue)
+            {
+                if (tableAttribute is SingleColumnTablePropertyAttribute)
+                {
+                    return new TableReferenceAlertProperty(propertyName, displayName, order.Next(), tableAttribute.ShowHeaders, propertyReferenceValue.ReferencePath);
+                }
+
+                // tableAttribute is MultiColumnTablePropertyAttribute
```

Comment:
>// tableAttribute is MultiColumnTablePropertyAttribute [](start = 16, length = 54)

Comment is wrong

Reply:
why do you say that?

---
In reply to: [279190841](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/61#discussion_r279190841) [](ancestors = 279190841)